WebService.Test/v1/Models/SimulationApiModel/MetricsApiModelTest.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using Microsoft.Azure.IoTSolutions.DeviceSimulation.Services.AzureManagementAdapter;
+using Microsoft.Azure.IoTSolutions.DeviceSimulation.WebService.v1.Models.SimulationApiModel;
+using WebService.Test.helpers;
+using Xunit;
+
+namespace WebService.Test.v1.Models.SimulationApiModel
+{
+    public class MetricsApiModelTest
+    {
+        [Fact, Trait(Constants.TYPE, Constants.UNIT_TEST)]
+        public void ItReturnsMetricsRequestsModelFromApiModel()
+        {
+            // Arrange
+            var apiModel = new MetricsRequestsApiModel();
```

Comment:
This test should add some requests to the API model, then confirm that they were converted properly.